### PR TITLE
Added slave monitoring

### PR DESCRIPTION
**Pull request by anderskvist**

### CODE REVIEW

File: plugins/agents/mysqlslave/Mysqlslave.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	for i, conn := range m.Connections {
+		points[i*2] = plugins.PointWithTag("mysqlslave.SecondsBehindMaster", conn.SecondsBehindMaster, "connection", conn.ConnectionName)
+		points[i*2+1] = plugins.PointWithTag("mysqlslave.ExecutedLogEntries", conn.ExecutedLogEntries, "connection", conn.ConnectionName)
+	}
+
+	return points
+}
+
+func (m *MysqlSlave) GetDoc() *plugins.Doc {
+	doc := plugins.NewDoc("MysqlSlave")
```
Comment (abrander) [inline plugins/agents/mysqlslave/Mysqlslave.go]: Could we change this to something like `doc := plugins.NewDoc("MySQL slave statistics")`?

File: plugins/agents/mysqlslave/Mysqlslave.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+	doc.AddTag("connection", "The connection name from MySQL")
+	doc.AddMeasurement("mysqlslave.SecondsBehindMaster", "Difference between the timestamp logged on the master for the event that the slave is currently processing, and the current timestamp on the slave. Zero if the slave is not currently processing an event.", "")
```
Comment (abrander) [inline plugins/agents/mysqlslave/Mysqlslave.go]: Could we add a unit here? I assume `"s"` would be correct.

File: plugins/agents/mysqlslave/Mysqlslave.go [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	doc.AddMeasurement("mysqlslave.ExecutedLogEntries", "How many log entries the slave has executed.", "")
```
Comment (abrander) [inline plugins/agents/mysqlslave/Mysqlslave.go]: I think `"n"` would be the correct unit here.

**Review by abrander — CHANGES_REQUESTED** [review]
Looks good besides a few minor details in `GetDoc()`.

**Comment by anderskvist** [comment]
@abrander all suggestions added

**Review by abrander — APPROVED** [review]

**Comment by abrander** [comment]
Thanks, merged :)

**Pull request merged**